spec/models/course_enrichment_spec.rb

```diff
@@ -57,4 +57,42 @@
       expect(CourseEnrichment.latest_first.last).to eq old_enrichment
     end
   end
+
+  describe '#unpublish' do
+    let(:provider) { create(:provider) }
+    let(:course) { create(:course, provider: provider) }
+    let(:last_published_timestamp_utc) { Date.new(2017, 1, 1) }
+    subject {
+      create(:course_enrichment, :published,
+        last_published_timestamp_utc: last_published_timestamp_utc,
+        course: course,
+        provider: provider)
+    }
+
+    context "to initial draft" do
```

Comment:
`context` should describe a new setup/initial state (fixtures/memoized values), not a new target state.

Reply:
fixed